aspnet-core/xml/Microsoft.AspNetCore.Mvc/ApiControllerAttribute.xml

```diff
@@ -55,10 +55,9 @@
             Controllers decorated with this attribute are configured with features and behavior targeted at improving the
             developer experience for building APIs.
             </para><para>
-            When decorated on an assembly, all controllers in the assembly will be treated as controllers with API behavior.
-            For more information, see <see href="https://docs.microsoft.com/aspnet/core/web-api/#apicontroller-attribute">ApiController attribute</see>.
```

Comment:
Revert this

Reply:
So this is a dupe I was talking about. It is also described in the `Remarks` sections